### PR TITLE
Fix a segfault in `new_empty_strided`

### LINKED ISSUE #82416: Segfault in `new_empty_strided`
### 🐛 Describe the bug

Looks like insufficient input validation. Segfaults with this repro:

```py
import torch

dtype = torch.float64
device = 'cpu'
pin_memory = False
layout = None

x = torch.tensor(-4.8270, dtype=torch.float64)
size = (2,)
stride = ()


res = x.new_empty_strided(
    size,
    stride,
    dtype=dtype,
    device=device,
    pin_memory=pin_memory,
    layout=layout,
)

print(res.stride())
print(res)
```

gdb core backtrace:
```cpp
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007fc2937e3d1f in at::detail::computeStorageNbytes(c10::ArrayRef<long>, c10::ArrayRef<long>, unsigned long, unsigned long) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
[Current thread is 1 (Thread 0x7fc2df7a4740 (LWP 1952527))]
warning: File "/home/nkaretnikov/pytorch/.gdbinit" auto-loading has been declined by your `auto-load safe-path' set to "$debugdir:$datadir/auto-load".
To enable execution of this file add
        add-auto-load-safe-path /home/nkaretnikov/pytorch/.gdbinit
line to your configuration file "/home/nkaretnikov/.gdbinit".
To completely disable this security protection add
        set auto-load safe-path /
line to your configuration file "/home/nkaretnikov/.gdbinit".
For more information about this security protection see the
"Auto-loading safe path" section in the GDB manual.  E.g., run from the shell:
        info "(gdb)Auto-loading safe path"
gef➤  bt
#0  0x00007fc2937e3d1f in at::detail::computeStorageNbytes(c10::ArrayRef<long>, c10::ArrayRef<long>, unsigned long, unsigned long) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#1  0x00007fc2937e4830 in at::detail::empty_strided_generic(c10::ArrayRef<long>, c10::ArrayRef<long>, c10::Allocator*, c10::DispatchKeySet, c10::ScalarType) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#2  0x00007fc2937e4cee in at::detail::empty_strided_cpu(c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ScalarType, bool) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#3  0x00007fc2937e4d83 in at::detail::empty_strided_cpu(c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#4  0x00007fc293d757ab in at::native::empty_strided_cpu(c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#5  0x00007fc294697764 in c10::impl::wrap_kernel_functor_unboxed_<c10::impl::detail::WrapFunctionIntoFunctor_<c10::CompileTimeFunctionPointer<at::Tensor (c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>), &at::(anonymous namespace)::(anonymous namespace)::wrapper__empty_strided>, at::Tensor, c10::guts::typelist::typelist<c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool> > >, at::Tensor (c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>)>::call(c10::OperatorKernel*, c10::DispatchKeySet, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#6  0x00007fc2943c0749 in at::_ops::empty_strided::redispatch(c10::DispatchKeySet, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#7  0x00007fc294669f1b in c10::impl::wrap_kernel_functor_unboxed_<c10::impl::detail::WrapFunctionIntoFunctor_<c10::CompileTimeFunctionPointer<at::Tensor (c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>), &at::(anonymous namespace)::empty_strided>, at::Tensor, c10::guts::typelist::typelist<c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool> > >, at::Tensor (c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>)>::call(c10::OperatorKernel*, c10::DispatchKeySet, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#8  0x00007fc294400adc in at::_ops::empty_strided::call(c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#9  0x00007fc293d762a8 in at::native::new_empty_strided(at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#10 0x00007fc2948bc7bd in c10::impl::wrap_kernel_functor_unboxed_<c10::impl::detail::WrapFunctionIntoFunctor_<c10::CompileTimeFunctionPointer<at::Tensor (at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>), &at::(anonymous namespace)::(anonymous namespace)::wrapper__new_empty_strided>, at::Tensor, c10::guts::typelist::typelist<at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool> > >, at::Tensor (at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>)>::call(c10::OperatorKernel*, c10::DispatchKeySet, at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#11 0x00007fc2943c0008 in at::_ops::new_empty_strided::redispatch(c10::DispatchKeySet, at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#12 0x00007fc295b1ef5e in c10::impl::wrap_kernel_functor_unboxed_<c10::impl::detail::WrapFunctionIntoFunctor_<c10::CompileTimeFunctionPointer<at::Tensor (c10::DispatchKeySet, at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>), &torch::autograd::VariableType::(anonymous namespace)::new_empty_strided>, at::Tensor, c10::guts::typelist::typelist<c10::DispatchKeySet, at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool> > >, at::Tensor (c10::DispatchKeySet, at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>)>::call(c10::OperatorKernel*, c10::DispatchKeySet, at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#13 0x00007fc2943ff14b in at::_ops::new_empty_strided::call(at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>, c10::optional<bool>) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_cpu.so
#14 0x00007fc29d3699a5 in torch::autograd::THPVariable_new_empty_strided(_object*, _object*, _object*) () from /home/nkaretnikov/pytorch/torch/lib/libtorch_python.so
#15 0x00005564e5ef0e7e in cfunction_call_varargs (kwargs=0x7fc2df3e9f80, args=0x7fc2df2c1d80, func=0x7fc1d33f9720) at /usr/local/src/conda/python-3.8.13/Objects/call.c:743
#16 PyCFunction_Call (func=0x7fc1d33f9720, args=0x7fc2df2c1d80, kwargs=0x7fc2df3e9f80) at /usr/local/src/conda/python-3.8.13/Objects/call.c:773
#17 0x00005564e5ed9631 in _PyObject_MakeTpCall (callable=0x7fc1d33f9720, args=<optimized out>, nargs=<optimized out>, keywords=<optimized out>) at /usr/local/src/conda/python-3.8.13/Python/errors.c:219
#18 0x00005564e5ed55df in _PyObject_Vectorcall (kwnames=0x7fc2df2b51d0, nargsf=<optimized out>, args=<optimized out>, callable=0x7fc1d33f9720) at /usr/local/src/conda/python-3.8.13/Include/cpython/abstract.h:125
#19 _PyObject_Vectorcall (kwnames=0x7fc2df2b51d0, nargsf=<optimized out>, args=<optimized out>, callable=0x7fc1d33f9720) at /usr/local/src/conda/python-3.8.13/Include/cpython/abstract.h:115
#20 call_function (kwnames=0x7fc2df2b51d0, oparg=<optimized out>, pp_stack=<synthetic pointer>, tstate=<optimized out>) at /usr/local/src/conda/python-3.8.13/Python/ceval.c:4963
#21 _PyEval_EvalFrameDefault (f=<optimized out>, throwflag=<optimized out>) at /usr/local/src/conda/python-3.8.13/Python/ceval.c:3515
#22 0x00005564e5ecf461 in PyEval_EvalFrameEx (throwflag=0x0, f=0x7fc2df3c9440) at /usr/local/src/conda/python-3.8.13/Python/ceval.c:741
#23 _PyEval_EvalCodeWithName (_co=<optimized out>, globals=<optimized out>, locals=<optimized out>, args=<optimized out>, argcount=<optimized out>, kwnames=0x0, kwargs=0x0, kwcount=<optimized out>, kwstep=0x2, defs=0x0, defcount=0x0, kwdefs=0x0, closure=0x0, name=0x0, qualname=0x0) at /usr/local/src/conda/python-3.8.13/Python/ceval.c:4298
#24 0x00005564e5f8ede9 in PyEval_EvalCodeEx (_co=<optimized out>, globals=<optimized out>, locals=<optimized out>, args=<optimized out>, argcount=<optimized out>, kws=<optimized out>, kwcount=0x0, defs=0x0, defcount=0x0, kwdefs=0x0, closure=0x0) at /usr/local/src/conda/python-3.8.13/Python/ceval.c:4327
#25 0x00005564e5f8edab in PyEval_EvalCode (co=<optimized out>, globals=<optimized out>, locals=<optimized out>) at /usr/local/src/conda/python-3.8.13/Python/ceval.c:718
#26 0x00005564e5faf903 in run_eval_code_obj (co=0x7fc2df3f7710, globals=0x7fc2df40ae00, locals=0x7fc2df40ae00) at /usr/local/src/conda/python-3.8.13/Python/pythonrun.c:1166
#27 0x00005564e5fae8e3 in run_mod (mod=<optimized out>, filename=<optimized out>, globals=0x7fc2df40ae00, locals=0x7fc2df40ae00, flags=<optimized out>, arena=<optimized out>) at /usr/local/src/conda/python-3.8.13/Python/pythonrun.c:1188
#28 0x00005564e5e4ef2f in pyrun_file (fp=0x5564e6cb0100, filename=0x7fc2df336db0, start=<optimized out>, globals=0x7fc2df40ae00, locals=0x7fc2df40ae00, closeit=0x1, flags=0x7ffc45d01288) at /usr/local/src/conda/python-3.8.13/Python/pythonrun.c:1085
#29 0x00005564e5e4ea23 in pyrun_simple_file (flags=0x7ffc45d01288, closeit=0x1, filename=0x7fc2df336db0, fp=0x5564e6cb0100) at /usr/local/src/conda/python-3.8.13/Python/pythonrun.c:439
#30 PyRun_SimpleFileExFlags (fp=0x5564e6cb0100, filename=<optimized out>, closeit=0x1, flags=0x7ffc45d01288) at /usr/local/src/conda/python-3.8.13/Python/pythonrun.c:472
#31 0x00005564e5e420ac in pymain_run_file (cf=0x7ffc45d01288, config=0x5564e6cb0f70) at /usr/local/src/conda/python-3.8.13/Modules/main.c:389
#32 pymain_run_python (exitcode=0x7ffc45d01280) at /usr/local/src/conda/python-3.8.13/Modules/main.c:614
#33 Py_RunMain () at /usr/local/src/conda/python-3.8.13/Modules/main.c:693
#34 0x00005564e5f83219 in Py_BytesMain (argc=<optimized out>, argv=<optimized out>) at /usr/local/src/conda/python-3.8.13/Modules/main.c:1113
#35 0x00007fc2df7cd0b3 in __libc_start_main (main=0x5564e5f831d0 <main>, argc=0x2, argv=0x7ffc45d01498, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7ffc45d01488) at ../csu/libc-start.c:308
#36 0x00005564e5f83125 in _start () at /usr/local/src/conda/python-3.8.13/Objects/frameobject.c:59
```

### Versions

git commit:

```
commit 1231194dd3e9290e418985755c89e68172d69015 (upstream-https/master)
Author: David Tsenter <dtsenter@fb.com>
Date:   Wed Jul 27 19:48:15 2022 +0000

    Add hardshrink op to metal backend (#82224)
    
    Summary:
    Add hardshrink op's metal implementation to pytorch codebase
    
    Heavily based on already existing Hardswish implementation
    
    Test Plan:
    Add two unit tests to compare the result of metal-based hardshrink function to that of the CPU implementation. Both tests pass:
    
    {F755783049}
    
    Differential Revision: D38152454
    
    Pull Request resolved: https://github.com/pytorch/pytorch/pull/82224
    Approved by: https://github.com/SS-JIA
```

collect:

```
Collecting environment information...
PyTorch version: 1.13.0a0+git1231194
Is debug build: False
CUDA used to build PyTorch: 11.4
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.4 LTS (x86_64)
GCC version: (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Clang version: Could not collect
CMake version: version 3.23.2
Libc version: glibc-2.31

Python version: 3.8.13 | packaged by conda-forge | (default, Mar 25 2022, 06:04:10)  [GCC 10.3.0] (64-bit runtime)
Python platform: Linux-5.4.0-104-generic-x86_64-with-glibc2.10
Is CUDA available: True
CUDA runtime version: 11.4.120
GPU models and configuration:
GPU 0: NVIDIA GeForce RTX 2060
GPU 1: NVIDIA GeForce RTX 2060

Nvidia driver version: 465.19.01
cuDNN version: Probably one of the following:
/usr/local/cuda-10.1.243/targets/x86_64-linux/lib/libcudnn.so.8
/usr/local/cuda-10.1.243/targets/x86_64-linux/lib/libcudnn_adv_infer.so.8
/usr/local/cuda-10.1.243/targets/x86_64-linux/lib/libcudnn_adv_train.so.8
/usr/local/cuda-10.1.243/targets/x86_64-linux/lib/libcudnn_cnn_infer.so.8
/usr/local/cuda-10.1.243/targets/x86_64-linux/lib/libcudnn_cnn_train.so.8
/usr/local/cuda-10.1.243/targets/x86_64-linux/lib/libcudnn_ops_infer.so.8
/usr/local/cuda-10.1.243/targets/x86_64-linux/lib/libcudnn_ops_train.so.8
/usr/local/cuda-10.2.89/targets/x86_64-linux/lib/libcudnn.so.8
/usr/local/cuda-10.2.89/targets/x86_64-linux/lib/libcudnn_adv_infer.so.8
/usr/local/cuda-10.2.89/targets/x86_64-linux/lib/libcudnn_adv_train.so.8
/usr/local/cuda-10.2.89/targets/x86_64-linux/lib/libcudnn_cnn_infer.so.8
/usr/local/cuda-10.2.89/targets/x86_64-linux/lib/libcudnn_cnn_train.so.8
/usr/local/cuda-10.2.89/targets/x86_64-linux/lib/libcudnn_ops_infer.so.8
/usr/local/cuda-10.2.89/targets/x86_64-linux/lib/libcudnn_ops_train.so.8
/usr/local/cuda-11.4.2/targets/x86_64-linux/lib/libcudnn.so.8
/usr/local/cuda-11.4.2/targets/x86_64-linux/lib/libcudnn_adv_infer.so.8
/usr/local/cuda-11.4.2/targets/x86_64-linux/lib/libcudnn_adv_train.so.8
/usr/local/cuda-11.4.2/targets/x86_64-linux/lib/libcudnn_cnn_infer.so.8
/usr/local/cuda-11.4.2/targets/x86_64-linux/lib/libcudnn_cnn_train.so.8
/usr/local/cuda-11.4.2/targets/x86_64-linux/lib/libcudnn_ops_infer.so.8
/usr/local/cuda-11.4.2/targets/x86_64-linux/lib/libcudnn_ops_train.so.8
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] mypy==0.960
[pip3] mypy-extensions==0.4.3
[pip3] numpy==1.21.6
[pip3] pytorch-sphinx-theme==0.0.24
[pip3] torch==1.13.0a0+git1231194
[conda] magma-cuda111             2.5.2                         1    pytorch
[conda] mkl                       2022.1.0           h84fe81f_915    conda-forge
[conda] mkl-include               2022.1.0           h84fe81f_915    conda-forge
[conda] numpy                     1.23.0           py38h3a7f9d9_0    conda-forge
[conda] pytorch-sphinx-theme      0.0.24                    dev_0    <develop>
[conda] torch                     1.13.0a0+git1231194           dev_0    <develop>

```

### PR DESCRIPTION
**Pull request by nkaretnikov**
Fixes #82416.

### CODE REVIEW

File: aten/src/ATen/EmptyTensor.cpp [hunk 1]
```diff
@@ -62,6 +62,11 @@ size_t computeStorageNbytes(
     size_t itemsize_bytes,
     size_t storage_offset
   ) {
+  // Similar to a message in as_strided_batching_rule
+  TORCH_CHECK(sizes.size() == strides.size(),
+    "size and stride must have the same length! Got size ", sizes,
+    " and stride ", strides);
```
Comment (peterbell10) [inline aten/src/ATen/EmptyTensor.cpp:68]: I looked at what happened previously, and I think this is the error message that was raised:
https://github.com/pytorch/pytorch/blob/34bdd46e6e6a15dfa1c765c1fb8d9d7bc02dc102/c10/core/TensorImpl.h#L1442-L1448

Comment (nkaretnikov) [reply aten/src/ATen/EmptyTensor.cpp:68]: yep, a similar message is in `new_empty_strided_batching_rule` as well. i'll change it to be consistent

**Comment by nkaretnikov** [comment]
ptal. the test failures look unrelated:

```
2022-07-28T20:56:02.9436210Z ======================================================================
2022-07-28T20:56:02.9436466Z FAIL [0.016s]: test_cat (__main__.TestFunctionalization)
2022-07-28T20:56:02.9436762Z ----------------------------------------------------------------------
2022-07-28T20:56:02.9437009Z Traceback (most recent call last):
2022-07-28T20:56:02.9437324Z   File "test_functionalization.py", line 376, in test_cat
2022-07-28T20:56:02.9437617Z     self.assertExpectedInline(reinplaced_logs, """\
2022-07-28T20:56:02.9438026Z   File "C:\Jenkins\Miniconda3\lib\site-packages\expecttest\__init__.py", line 262, in assertExpectedInline
2022-07-28T20:56:02.9438432Z     self.assertMultiLineEqualMaybeCppStack(expect, actual, msg=help_text)
2022-07-28T20:56:02.9438915Z   File "C:\Jenkins\Miniconda3\lib\site-packages\expecttest\__init__.py", line 281, in assertMultiLineEqualMaybeCppStack
2022-07-28T20:56:02.9439304Z     self.assertMultiLineEqual(expect, actual[:len(expect)], *args, **kwargs)
2022-07-28T20:56:02.9439723Z AssertionError: "\n\n[55 chars]mpty.SymInt([0], dtype = torch.float32, device[134 chars]    " != "\n\n[55 chars]mpty.memory_format([0], dtype = torch.float32,[133 chars]efau"
2022-07-28T20:56:02.9440026Z   
2022-07-28T20:56:02.9440150Z   
2022-07-28T20:56:02.9440284Z   
2022-07-28T20:56:02.9440440Z   def forward(self, a_1):
2022-07-28T20:56:02.9440741Z -     empty = torch.ops.aten.empty.SymInt([0], dtype = torch.float32, device = device(type='cpu'), pin_memory = False)
2022-07-28T20:56:02.9441030Z ?                                  ^  ^^
2022-07-28T20:56:02.9441340Z +     empty = torch.ops.aten.empty.memory_format([0], dtype = torch.float32, device = device(type='cpu'), pin_memory = False)
2022-07-28T20:56:02.9441635Z ?                                  ^^^^^ ++++ ^
2022-07-28T20:56:02.9441867Z       cat_default = torch.ops.aten.cat.default([a_1]);  a_1 = None
2022-07-28T20:56:02.9442099Z -     return cat_default
2022-07-28T20:56:02.9442274Z ?                     ---
2022-07-28T20:56:02.9442605Z +     return cat_defau-      : To accept the new output, re-run test with envvar EXPECTTEST_ACCEPT=1 (we recommend staging/committing your changes before doing this)
2022-07-28T20:56:02.9442860Z 
2022-07-28T20:56:02.9442956Z ======================================================================
2022-07-28T20:56:02.9443200Z FAIL [0.016s]: test_multi_out (__main__.TestFunctionalization)
2022-07-28T20:56:02.9443490Z ----------------------------------------------------------------------
2022-07-28T20:56:02.9444394Z Traceback (most recent call last):
2022-07-28T20:56:02.9444711Z   File "test_functionalization.py", line 209, in test_multi_out
2022-07-28T20:56:02.9445002Z     self.assertExpectedInline(reinplaced_logs, """\
2022-07-28T20:56:02.9445398Z   File "C:\Jenkins\Miniconda3\lib\site-packages\expecttest\__init__.py", line 262, in assertExpectedInline
2022-07-28T20:56:02.9445786Z     self.assertMultiLineEqualMaybeCppStack(expect, actual, msg=help_text)
2022-07-28T20:56:02.9446316Z   File "C:\Jenkins\Miniconda3\lib\site-packages\expecttest\__init__.py", line 281, in assertMultiLineEqualMaybeCppStack
2022-07-28T20:56:02.9446725Z     self.assertMultiLineEqual(expect, actual[:len(expect)], *args, **kwargs)
2022-07-28T20:56:02.9447152Z AssertionError: "\n\n[55 chars]mpty.SymInt([4], dtype = torch.float32, device[360 chars]    " != "\n\n[55 chars]mpty.memory_format([4], dtype = torch.float32,[359 chars]etur"
2022-07-28T20:56:02.9447679Z Diff is 929 characters long. Set self.maxDiff to None to see it. : To accept the new output, re-run test with envvar EXPECTTEST_ACCEPT=1 (we recommend staging/committing your changes before doing this)
2022-07-28T20:56:02.9448048Z 
2022-07-28T20:56:02.9534112Z ----------------------------------------------------------------------
2022-07-28T20:56:02.9534367Z Ran 39 tests in 1.719s
2022-07-28T20:56:02.9534479Z 
2022-07-28T20:56:02.9534599Z FAILED (failures=2, skipped=1, expected failures=6)
```

**Review by peterbell10 — APPROVED** [review]

**Review by ngimel — APPROVED** [review]

**Comment by ngimel** [comment]
@nkaretnikov can you please rebase to see if error goes away?

**Comment by nkaretnikov** [comment]
@ngimel rebased on a recent commit in `viable/strict` with no failures (as per hud), ci is green now

**Pull request closed without merging**